core/templates/pages/release-coordinator-page/features-tab/features-tab.component.spec.ts

```diff
@@ -313,6 +314,91 @@ describe('Release coordinator page feature tab', function() {
     });
   });
 
+  describe('.getFeatureValidOnCurrentServer', () => {
+    let featureFlagDevStage = PlatformParameter.createFromBackendDict({
+      data_type: 'bool',
+      default_value: false,
+      description: 'This is a dummy feature flag.',
+      feature_stage: FeatureStage.DEV,
+      is_feature: true,
+      name: 'dummy_feature',
+      rule_schema_version: 1,
+      rules: [{
+        filters: [
+          {
+            type: PlatformParameterFilterType.ServerMode,
+            conditions: [['=', ServerMode.Dev]]
+          }
+        ],
+        value_when_matched: true,
+      }],
+    });
+
+    let featureFlagProdStage = PlatformParameter.createFromBackendDict({
+      data_type: 'bool',
+      default_value: false,
+      description: 'This is a dummy feature flag.',
+      feature_stage: FeatureStage.PROD,
+      is_feature: true,
+      name: 'dummy_feature',
+      rule_schema_version: 1,
+      rules: [{
+        filters: [
+          {
+            type: PlatformParameterFilterType.ServerMode,
+            conditions: [['=', ServerMode.Dev]]
+          }
+        ],
+        value_when_matched: true,
+      }],
+    });
+
+    afterEach(() => {
+      component.serverStage = '';
+    });
+
+    it('should return true when the server in dev stage and feature ' +
+    'stage is dev too', (() => {
+      component.serverStage = 'dev';
+      expect(component.getFeatureValidOnCurrentServer(
+        featureFlagDevStage)).toBe(true);
+    }));
+
+    it('should return false when the server in test stage and feature ' +
+    'stage is dev', (() => {
+      component.serverStage = 'test';
+      expect(component.getFeatureValidOnCurrentServer(
+        featureFlagDevStage)).toBe(false);
```

Comment:
```suggestion
      component.serverStage = 'test';
 
      expect(component.getFeatureValidOnCurrentServer(
        featureFlagDevStage)).toBe(false);
```

Please make the same change for the remaining tests. Thanks!

Reply:
Done